internal/gengapic/lro.go

```diff
@@ -280,6 +281,17 @@ func (g *generator) lroType(servName string, serv *descriptor.ServiceDescriptorP
 	return nil
 }
 
-func lroTypeName(methodName string) string {
-	return methodName + "Operation"
+func lroTypeName(m *descriptor.MethodDescriptorProto) string {
+	// This whole if block is a hack to workaround a operation handler namespace
+	// collision. We should remove this in the future if the design is fixed for
+	// the v1 api. This is for aiplatform.featureregistryservice.createfeature.
+	if eHTTP, ok := proto.GetExtension(m.GetOptions(), annotations.E_Http).(*annotations.HttpRule); ok && eHTTP != nil && eHTTP.Pattern != nil {
+		switch t := eHTTP.Pattern.(type) {
+		case *annotations.HttpRule_Post:
+			if t.Post == "/v1beta1/{parent=projects/*/locations/*/featureGroups/*}/features" {
```

Comment:
It seems at least a tiny bit possible that someday another RPC will match this pattern...

Reply:
It may, but I believe it is unlikely. Ideally this code gets deleted in the next couple of months. If we end up having to do this for the v1 service we can make the logic tighter at that time. I was trying to avoid passing a lot more info throughout the stack if possible so this is easy to unwind.